config/packages/fos_http_cache.yaml

```diff
@@ -9,7 +12,7 @@ fos_http_cache:
     proxy_client:
         varnish:
             http:
-                servers: "%cache_servers%"
+                servers: ["localhost"]
```

Comment:
why like that? Personally i have varnish on other host than localhost.

Reply:
it's still WIP, hardcoded it to make it work for now